services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/api_client/_public.py

```diff
@@ -355,6 +362,21 @@ async def restore_service_state(self, dynamic_sidecar_endpoint: AnyHttpUrl) -> N
             _debug_progress_callback,
         )
 
+    async def pull_user_services_images(
+        self, dynamic_sidecar_endpoint: AnyHttpUrl
+    ) -> None:
+        response = await self._thin_client.post_containers_user_services_images_pull(
+            dynamic_sidecar_endpoint
+        )
+        task_id: TaskId = response.json()
+
+        await self._await_for_result(
+            task_id,
+            dynamic_sidecar_endpoint,
+            self._dynamic_services_scheduler_settings.DYNAMIC_SIDECAR_API_USER_SERVICES_PULLING_TIMEOUT,
+            _debug_progress_callback,
```

Comment:
also wondering about that callback. this is supposed to come all the way to the frontend via the adequate message. is this still the case?

Reply:
It still sends some messages to the logger, but these do not contain the actual progress, just some strings.
We might want to drop this system or rename/refactor to make it more clear what it does.